### PR TITLE
update l/r/s/a with more details

### PR DESCRIPTION
**Pull request by jamesmontemagno**

### CODE REVIEW

File: release-notes/9.0/preview/preview6/aspnetcore.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 
-- [Fingerprinting of static web assets](#fingerprinting-of-static-web-assets)
-- [Improved distributed tracing for SignalR](#improved-distributed-tracing-for-signalr)
-- [Enhancements to Microsoft.AspNetCore.OpenAPI](#enhancements-to-microsoftaspnetcoreopenapi)
-- [Analyzer to warn when `[Authorize]` is overridden by `[AllowAnonymous]`](#analyzer-to-warn-when-authorize-is-overridden-by-allowanonymous-from-farther-away)
+- [Fingerprinting of static web assets](#fingerprinting-of-static-web-assets) ensuring that stale assets aren't used and enables improved caching behavior for faster load time
+- [Improved distributed tracing for SignalR](#improved-distributed-tracing-for-signalr) with a new `ActivitySource`
+- [Enhancements to Microsoft.AspNetCore.OpenAPI](#enhancements-to-microsoftaspnetcoreopenapi) including [completion enhancements(#completion-enhancements-and-package-install-recommendations-for-openapi-package)], support for [`[Required]` and `[DefaultValue]` attributes](#support-for-required-and-defaultvalue-attributes-on-parameters-or-properties), [schema transforms](#support-for-schema-transformers-on-openapi-document) on OpenAPI documents,
```
Comment (jongalloway) [inline release-notes/9.0/preview/preview6/aspnetcore.md:7]:
```suggestion
- [Enhancements to Microsoft.AspNetCore.OpenAPI](#enhancements-to-microsoftaspnetcoreopenapi) including [completion enhancements](#completion-enhancements-and-package-install-recommendations-for-openapi-package)], support for [`[Required]` and `[DefaultValue]` attributes](#support-for-required-and-defaultvalue-attributes-on-parameters-or-properties), [schema transforms](#support-for-schema-transformers-on-openapi-document) on OpenAPI documents, 
```

fix link on completion enhancements

File: release-notes/9.0/preview/preview6/libraries.md [hunk 2]
```diff
@@ -4,19 +4,19 @@ Here's a summary of what's new in .NET Libraries in this preview release:
...
 - [Improvements to System.Numerics](#improvements-to-systemnumerics)
 - [Support Primary Constructors in Logging Source Generator](#support-primary-constructors-in-logging-source-generator)
-- [System.Text.Json](#systemtextjson)
-- [`[GeneratedRegex]` on properties](#generatedregex-on-properties)
-- [`Regex.EnumerateSplits`](#regexenumeratesplits)
-- [`OrderedDictionary<TKey, TValue>`](#ordereddictionarytkey-tvalue)
-- [`ReadOnlySet<T>`](#readonlysett)
+- [System.Text.Json enhancements](#systemtextjson) includinga a new [JsonSchemaExporter](#jsonschemaexporter), [nullable annotations recognition](#respecting-nullable-annotations), [requiring non-optionsl constructor parameters](#requiring-non-optional-constructor-parameters), [ordering `JsonObject` properties](#ordering-jsonobject-properties), and new [contract metadata APIs](#additional-contract-metadata-apis)
+- [`[GeneratedRegex]` can now be used on properties](#generatedregex-on-properties)
+- New [`EnumerateSplits`](#regexenumeratesplits) method for `Regex` to split more colleciton types.
```
Comment (jongalloway) [inline release-notes/9.0/preview/preview6/libraries.md:9]:
```suggestion
- New [`EnumerateSplits`](#regexenumeratesplits) method for `Regex` to split more colleciton types
```
standardize on . or not at the end of bullets

File: release-notes/9.0/preview/preview6/runtime.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
 
-- [ARM64 Code Generation](#arm64-code-generation)
-- [Code Layout](#code-layout)
-- [Loop Optimizations](#loop-optimizations)
-- [Reduced Address Exposure](#reduced-address-exposure)
-- [AVX10v1 Support](#avx10v1-support)
+- [ARM64 Code Generation](#arm64-code-generation) now adds ability to store operations
+- [Code Layout](#code-layout) - RyuJIT's block reordering algorithm with a simpler, more global approach
+- [Loop Optimizations](#loop-optimizations) for code size reduction and performance improvements
+- [Reduced Address Exposure](#reduced-address-exposure) through RyuJIT improvements to better track usagae of local variable addreses.
```
Comment (jongalloway) [inline release-notes/9.0/preview/preview6/runtime.md:8]:
```suggestion
- [Reduced Address Exposure](#reduced-address-exposure) through RyuJIT improvements to better track usage of local variable addreses
```

typo, standardize on . or not at the end of bullets

File: release-notes/9.0/preview/preview6/libraries.md [hunk 4]
```diff
@@ -4,19 +4,19 @@ Here's a summary of what's new in .NET Libraries in this preview release:
...
 - [Improvements to System.Numerics](#improvements-to-systemnumerics)
 - [Support Primary Constructors in Logging Source Generator](#support-primary-constructors-in-logging-source-generator)
-- [System.Text.Json](#systemtextjson)
-- [`[GeneratedRegex]` on properties](#generatedregex-on-properties)
-- [`Regex.EnumerateSplits`](#regexenumeratesplits)
-- [`OrderedDictionary<TKey, TValue>`](#ordereddictionarytkey-tvalue)
-- [`ReadOnlySet<T>`](#readonlysett)
+- [System.Text.Json enhancements](#systemtextjson) includinga a new [JsonSchemaExporter](#jsonschemaexporter), [nullable annotations recognition](#respecting-nullable-annotations), [requiring non-optionsl constructor parameters](#requiring-non-optional-constructor-parameters), [ordering `JsonObject` properties](#ordering-jsonobject-properties), and new [contract metadata APIs](#additional-contract-metadata-apis)
+- [`[GeneratedRegex]` can now be used on properties](#generatedregex-on-properties)
+- New [`EnumerateSplits`](#regexenumeratesplits) method for `Regex` to split more colleciton types
```
Comment (richlander) [inline release-notes/9.0/preview/preview6/libraries.md:9]:
```suggestion
- New [`EnumerateSplits`](#regexenumeratesplits) method for `Regex` to split more collection types
```

File: release-notes/9.0/preview/preview6/libraries.md [hunk 5]
```diff
@@ -4,19 +4,19 @@ Here's a summary of what's new in .NET Libraries in this preview release:
...
+- Introduction of generic `OrderedDictionary` with [`OrderedDictionary<TKey, TValue>`](#ordereddictionarytkey-tvalue) 
+- New [`ReadOnlySet<T>`](#readonlysett) when needing to create a read-only warapper around `ISet<T>`
```
Comment (richlander) [inline release-notes/9.0/preview/preview6/libraries.md:11]:
```suggestion
- New [`ReadOnlySet<T>`](#readonlysett) when needing to create a read-only wrapper around `ISet<T>`
```

File: release-notes/9.0/preview/preview6/libraries.md [hunk 6]
```diff
@@ -4,19 +4,19 @@ Here's a summary of what's new in .NET Libraries in this preview release:
...
 - [`allows ref struct` used in many places throughout the libraries](#allows-ref-struct-used-in-many-places-throughout-the-libraries)
 - [Collection lookups with spans](#collection-lookups-with-spans)
-- [More span-based APIs](#more-span-based-apis)
-- [Base64Url](#base64url)
+- [More span-based APIs](#more-span-based-apis) including `StartsWith` and `EndsWith` extension methods
+- [Base64Url](#base64url) for optimzed enconding and decoding
```
Comment (richlander) [inline release-notes/9.0/preview/preview6/libraries.md:15]:
```suggestion
- [Base64Url](#base64url) for optimzed encoding and decoding
```

File: release-notes/9.0/preview/preview6/libraries.md [hunk 7]
```diff
@@ -4,19 +4,19 @@ Here's a summary of what's new in .NET Libraries in this preview release:
 
 - [Improvements to System.Numerics](#improvements-to-systemnumerics)
 - [Support Primary Constructors in Logging Source Generator](#support-primary-constructors-in-logging-source-generator)
-- [System.Text.Json](#systemtextjson)
-- [`[GeneratedRegex]` on properties](#generatedregex-on-properties)
-- [`Regex.EnumerateSplits`](#regexenumeratesplits)
-- [`OrderedDictionary<TKey, TValue>`](#ordereddictionarytkey-tvalue)
-- [`ReadOnlySet<T>`](#readonlysett)
+- [System.Text.Json enhancements](#systemtextjson) includinga a new [JsonSchemaExporter](#jsonschemaexporter), [nullable annotations recognition](#respecting-nullable-annotations), [requiring non-optionsl constructor parameters](#requiring-non-optional-constructor-parameters), [ordering `JsonObject` properties](#ordering-jsonobject-properties), and new [contract metadata APIs](#additional-contract-metadata-apis)
```
Comment (jamesmontemagno) [inline release-notes/9.0/preview/preview6/libraries.md:7]:
```suggestion
- [System.Text.Json enhancements](#systemtextjson) including a new [JsonSchemaExporter](#jsonschemaexporter), [nullable annotations recognition](#respecting-nullable-annotations), [requiring non-optionsl constructor parameters](#requiring-non-optional-constructor-parameters), [ordering `JsonObject` properties](#ordering-jsonobject-properties), and new [contract metadata APIs](#additional-contract-metadata-apis)
```

**Review by richlander — APPROVED** [review]

**Review by Jjajsja — APPROVED** [review]

**Review by Jjajsja — APPROVED** [review]

**Pull request merged**